app/models/manageiq/providers/amazon/inventory/parser/cloud_manager.rb

```diff
@@ -215,7 +215,7 @@ def cloud_databases
       persister.cloud_databases.build(
         :ems_ref               => cloud_database["dbi_resource_id"],
         :name                  => cloud_database["db_instance_identifier"],
-        :status                => cloud_database["db_instance_status"],
+        :status                => normalize_db_status(cloud_database["db_instance_status"]),
```

Comment:
should we store the raw status somewhere as well?

Reply:
Yes but in order to backport this I want to merge the normalized status first (which is what the UI is currently calling), backport it, then add a schema migration to add the raw_status in a follow-up